metricbeat/module/couchbase/node/_meta/fields.yml

```diff
@@ -0,0 +1,97 @@
+- name: node
+  type: group
+  description: >
+    node
+  fields:
+    - name: hostname
+      type: keyword
+      description: >
+        The hostname of the Node
+    - name: uptime
+      type: keyword
+      description: >
+        Time during which the node was in operation (sec)
+    - name: mcd_memory.allocated.bytes
+      type: long
+      description: >
+        Amount of memcached memory allocated (bytes)
+    - name: mcd_memory.reserved.bytes
+      type: long
+      description: >
+        Amount of memcached memory reserved (bytes)
+    - name: cmd_get
+      type: long
+      description: >
+        Number of get commands
+    - name: cpu_utilization_rate.pct
+      type: float
+      description: >
+        The CPU utilization rate (%)
+    - name: couch_docs_actual_disk_size
+      type: long
+      description: >
+        Amount of disk space used by Couch docs.(bytes)
+    - name: couch_docs_data_size.bytes
+      type: long
+      description: >
+        Data size of couch documents associated with a node (bytes)
+    - name: couch_spatial_data_size.bytes
+      type: long
+      description: >
+        couchSpatialDataSize field
+    - name: couch_spatial_disk_size.bytes
```

Comment:
couch_spatial.

I'm tempted to say even `couch.spatial.*` and apply the same to the above as there are different namespaces inside couch it seems.